src/upgrades/v0_9_0/pubd_objects_migration.rs

```diff
@@ -173,31 +191,46 @@ impl UpgradeStore for PubdStoreMigration {
                 );
             }
 
+            // We can skip all publish commands - they are no longer used. State is kept in
+            // the RepositoryObjects structure instead.
             if old_cmd_key.name().contains("pubd-publish.json") {
                 continue; // There is no migration needed for these commands.
             }
 
+            // Read and parse the old command.
             let mut old_cmd: OldStoredRepositoryCommand = self.get(&old_cmd_key)?;
 
+            // If the command was a success, then it will have events. Successful commands
+            // that resulted in no changes are simply not recorded. That said, it may turn
+            // out that the event list to migrate is empty - in that case we skip this command
+            // and continue the command loop.
+            //
+            // Note that if the command resulted in an error we do not not get 'Some' empty
+            // vec of events, but we get a None. So such commands will be migrated.
```

Comment:
Same comment about None as above, as None isn't handled here is it correct to say "will be migrated" instead of "will NOT be migrated" ?